container/README.md

```diff
@@ -0,0 +1,65 @@
+# Holochain Rust Container
+
+[![Project](https://img.shields.io/badge/project-holochain-blue.svg?style=flat-square)](http://holochain.org/)
+[![PM](https://img.shields.io/badge/pm-waffle-blue.svg?style=flat-square)](https://waffle.io/holochain/org)
+[![Chat](https://img.shields.io/badge/chat-chat%2eholochain%2enet-blue.svg?style=flat-square)](https://chat.holochain.net)
+
+This crate implements a reference container for serving Holochain DNAs.
+
+## Install
+
+Our recommended pattern for the installation of the container is to download the binary for your platform from our [releases](https://github.com/holochain/holochain-rust/releases) page. Otherwise, you can proceed with the more complex instructions for building from source, below.
+
+### Building From Source
+
+These dependencies need to be installed in order to compile, and use `holochain_container`:
+
+- [Rust](https://www.rust-lang.org/en-US/install.html)
+  - needs to be the `nightly` build, so use the following commands, once you have first installed Rust
+  - `rustup toolchain install nightly`
```

Comment:
is an unpinned nightly safe?

Reply:
probably not for ever, but this is also what's in the cmd README so I'm just going to flag this for followup. and I think it should work for now...